torch/csrc/jit/node_hashing.cpp

```diff
@@ -133,7 +133,7 @@ size_t HashNode::operator()(const Node* k) const {
 bool EqualNode::operator()(const Node* lhs, const Node* rhs) const {
   if (lhs == nullptr && rhs == nullptr)
     return true;
-  if (lhs == nullptr || rhs == nullptr)
+  if (lhs == nullptr ^ rhs == nullptr)
```

Comment:
I'll note that the condition where they are both `nullptr` is checked above.

Reply:
Thanks for your help😊